styles/style.css

```diff
@@ -42,6 +42,97 @@
     justify-content: space-between;
     background: white;
     box-shadow: 0 2px 8px #d0d9d4;
+    transition: all 1s ease-out;
+  }
+
+  .mobile_menu {
+    display: none;
+    position: fixed;
+    top: 0;
+    z-index: 11;
+    height: 90vh;
+    width: 100vw;
+    background: white;
+    border-radius: 0 0 30px 30px;
+    align-items: center;
+    transition: all 1s ease-out;
+    flex-direction: column;
+    justify-content: flex-start;
+    box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
+  }
+
+  @keyframes turn_go {
+    0% {
+      transform: translateX(100vw);
+    }
+
+    100% {
+      transform: translateX(0);
+    }
+  }
+
+  @keyframes turn_back {
+    0% {
+      transform: translateX(-100vw);
+    }
+
+    100% {
+      transform: translateX(0);
+    }
+  }
+
+  .d_none {
+    left: 101vw;
+    display: flex;
+  }
+
+  .d_flex {
+    left: 0;
+    display: flex;
+    animation: turn_go 2s;
+  }
+
+  .m_head {
+    width: 100%;
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 20px 30px;
+  }
+
+  #close_menu_icon {
+    width: 100%;
+    height: 100%;
+  }
+
+  #close_pop {
+    width: 52px;
+    height: 55px;
+  }
+
+  .mobile_menu .navigation {
+    /* background: gray; */
+  }
```

Comment:
I suggest you remove this block in your CSS file if you don't need it, to keep your code clean and look more professional. 😄

Reply:
Ok done with all comments and console.log instructions. 
But @stanleySimeon I think the readme.md file is in the project and in this branch with the required demo link.